src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,23 +21,18 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        String reverse;
-        String wholeStr;
         String[] tokens = text.toLowerCase()
                 .replaceAll("[,.:;()!?]", "")
-                .replaceAll("(?<=[ ])[-](?=[ ])", "")
-                .replaceAll(" {2,}", " ")
```

Comment:
you can do that with one replaceAll() try to find regex for it

Reply:
Fixed